.github/workflows/release.yaml

```diff
@@ -0,0 +1,514 @@
+name: Build and release
```

Comment:
Please add a high level description of what this file contains. I think I can tell, it basically just sets up all the docker runners for each platform we support. 

What's the maintenance of this? Just checking distrowatch every couple of weeks whether some major platform has an update? Can we just pull the images from dockerhub, add the dependencies we think we need and then add the new image here?